lib/src/cli/cli_runner.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:architecture_linter/src/cli/commands/analyze_command.dart';
+import 'package:architecture_linter/src/cli/printer/printer.dart';
+import 'package:args/command_runner.dart';
+
+class CliRunner extends CommandRunner<void> {
+  CliRunner() : super("analyzer", "Analyze your project architecture") {
+    addCommand(AnalyzeCommand(printer));
+  }
+
+  final Printer printer = Printer(stdout);
+
+  @override
+  Future<void> run(Iterable<String> args) async {
+    try {
+      await super.run(args);
+    } on UsageException catch (e) {
+      printer
+        ..write(e.message)
+        ..write(e.usage);
+
+      exit(64);
```

Comment:
Can you name 1 and 64 in order to developer reading code will know what they exactly mean?